auth/cloud/aws.go

```diff
@@ -0,0 +1,82 @@
+package cloud
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/banzaicloud/pipeline/config"
+	"github.com/banzaicloud/pipeline/constants"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultRegion = "eu-west-1"
+)
+
+var logger *logrus.Logger
+var log *logrus.Entry
+
+// Simple init for logging
+func init() {
+	logger = config.Logger()
+	log = logger.WithFields(logrus.Fields{"action": "Cluster"})
+}
+
+// awsVerify for validation AWS credentials
+type awsVerify struct {
```

Comment:
Why we need this struct? You are using it in `CreateAWSSecret` as a return type but inside `CreateEC2Client` you are not using it, also in case of `CreateAWSCredentials` you are returning `credentials *credentials.Credentials` instead of awsVerify:)